components/dialog/index.css

```diff
@@ -344,7 +345,6 @@
 			auto
 			var(--mod-standard-dialog-spacing-grid-padding, var(--spectrum-standard-dialog-spacing-grid-padding));
 
-		/* TODO: investigate if we could refactor the grid to make the footer text/checkbox stack on top of the button group */
```

Comment:
pretty sure it's already doing that. Maybe this was a comment from the standard dialog migration, that I fixed in the takeover dialog migration.